### ISSUE TITLE
Vehicle Handling Click to Roll

### ISSUE THREAD

**Issue by Astrotech89** [Astrotech89/ME5e-FoundryVTT#4]
Old ME5e module had slot for Vehicle handling but was unable to add click to roll feature and proficiencies on character sheet

**Comment by bushongj** [comment]
I've looked a little into this. So far, this appears to be a limitation with how modules alter the game system. If we want vehicle handling to be a clickable function on the character sheet, we will need to understand how to add macros to a character sheet. There is nothing we can leverage from the DnD5e game system to do this as far as I know.

**Comment by sparkcity** [comment]
Yes, this is a limitation that has been a thorn in my side for some time. Modules can't add anything to the Actor model which is what I was attempting to do here with adding "newskills" to the Actor model and tried several other ways (none successful):

> function extendActorData() {
> 	const dat = this.data.data;
> 	dat["newskills"] = dat["newskills"] || 
> 	{ "veh": 
> 		{
> 			value: ''
> 		}
> 	};

My other thought would be that maybe this information can be stored in a separate json, updated from the sheet, and called as needed? I would have to experiment to get a proof of concept first. This saved information would also need to be deleted when he associated actor was deleted so it's not leaving all these orphans taking up space.

**Comment by Astrotech89** [comment]
Yes, this will need to be a project for when we work on a game system rather than a module.
For now, users will have to track it or we could just replace animal handling with vehicle handling knowing the modifier is different.

**Comment by marxpark00** [comment]
i used a module for manage this, so maybe i can study that module and try to replicate here

**Comment by marxpark00** [comment]
the module is this https://foundryvtt.com/packages/skill-customization-5e and i added in the module of mass effect this 
![imagen](https://user-images.githubusercontent.com/100231515/156019711-f9f84fab-5dbe-4af1-82bb-b8fbdc7f191e.png)
so i can later modify the bonus in the sheet like this=>
![imagen](https://user-images.githubusercontent.com/100231515/156019925-84f0c5c5-d322-4d5f-bbc1-11e175a450af.png)

**Comment by sparkcity** [comment]
For a minimum viable product version of the ME5e module, it's certainly an option to add the skill customization module as a dependency as a temporary quickfix.
